app/services/EssttpBarsService.scala

```diff
@@ -63,16 +63,21 @@ class EssttpBarsService @Inject() (
       )
       .flatMap { result =>
         auditService.auditBarsCheck(journey, bankDetails, typeOfBankAccount, result)
-        barsVerifyStatusConnector
-          .update(taxId)
-          .map { verifyStatus =>
-            // here we catch a lockout BarsStatus condition,
-            // and force a TooManyAttempts (BarsError) response
-            verifyStatus.lockoutExpiryDateTime
-              .fold(result) { expiry =>
-                result.flatMap(resp => Left(TooManyAttempts(resp, expiry)))
+        result match {
+          case Left(_: BarsValidateError) | Left(_: SortCodeOnDenyListValidateError) =>
+            Future.successful(result) // don't update the verify count on validate errors
```

Comment:
are there any unit tests to test the new behaviour?

Reply:
There will be: https://jira.tools.tax.service.gov.uk/browse/SUPP-483